### PR TITLE
Add note to rebalancing

### PR DESCRIPTION
**Pull request by Zelldon**
## What is the purpose of the change

_Briefly describe the change you are making, this helps the reviewer by providing an overview._

Extended the note to highlight an API we have, which can be used for rebalancing leadership.

## Are there related marketing activities

_Are there any marketing activities related to the related feature or component? If so, please briefly describe them._

## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/zeebe/technical-concepts/clustering.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ Each broker in the cluster may be both leader and follower at the same time for
 There is no active load balancing across partitions. Each leader election for any partition is autonomous and independent of leader elections for other partitions.
 
 This may lead to one node becoming the leader for all partitions. This is not a problem for fault tolerance as the guarantees of replication remain. However, this may negatively impact throughput as all traffic hits one node.
+
+To reach a well-distributed leadership again the [Rebalancing API](https://docs.camunda.io/docs/next/self-managed/zeebe-deployment/operations/rebalancing/) can be used. Be aware that this is on best effort basis.
```
Comment (oleschoenburg) [inline docs/components/zeebe/technical-concepts/clustering.md:42]: :upside_down_face: This should probably be a relative link, not hardcoded to `docs.camunda.io`.

**Review by oleschoenburg — APPROVED** [review]
Nice, good note to have :+1: 

I guess this can also be applied to 8.0 and 8.1

File: docs/components/zeebe/technical-concepts/clustering.md (hunk 1 above)
Comment (Zelldon) [reply docs/components/zeebe/technical-concepts/clustering.md:42]: Upsi 😃 you're right. I was to quick.

**Review by christinaausley — APPROVED** [review]
Technical review in latest commit, and I added this note to the previous versions as well 👍

**Pull request merged**